.github/ISSUE_TEMPLATE/application.yml

```
@@ -0,0 +1,160 @@
+
+        Once you submit your application our friendly 1Password Bot will perform an initial review, followed by a human who will do a final pass. Upon approval we'll apply a non-expiring promotion to your account, and add your application details to this repository. Learn more about [membership details.](https://github.com/1Password/1password-teams-open-source#-membership-details)
+
+        If you're not sure if you're eligible, or have any questions before applying, please contact [opensource@1password.com](mailto:opensource@1password.com).
+
+        ---
+
+  - type: input
+    attributes:
+      label: Account URL
+      description: |
+        The full URL for your 1Password Teams account. Don't have an account? [Start a 14 day trial.](https://start.1password.com/signup/?t=B)
+      placeholder: example.1password.com
+    validations:
+      required: true
+
+  - type: checkboxes
+    attributes:
+      label: Non-commercial confirmation
+      description: The 1Password account joining this program must not be associated with commercial activity.
+      options:
+        - label: No, this account will not be used for commercial activity
+
+  - type: checkboxes
+    attributes:
+      label: Team application
+      description: Is this application for an open source team (and not a specific project)?
+      options:
+        - label: Yes, this application is for a team
+
+  - type: checkboxes
+    attributes:
+      label: Event application
+      description: Is this application for an open source meetup, event, or conference?
+      options:
+        - label: Yes, this application is for an event
+
+  - type: markdown
+    attributes:
+      value: |
+        ## 📦 Your project
+
+        Let's see what you're working on.
+
+  - type: input
+    attributes:
+      label: Project name
+      description: The name of your open source project.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Short description
+      description: A short description of your project, team, or event. Try to keep it to 50 words or less. Formatting will be stripped, and emojis are not allowed.
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Number of team members/core contributors
+      description: This may change over time, but will help us understand the initial size of the team for this account.
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Homepage URL
+      description: |
+        Where can we learn more about your project, team, or event? This could be the official website, a repository, a profile, or an event page.
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Repository URL
+      description: Can be left blank if the same as the Homepage URL. If part of a team, pick your favourite repo!
```

🇺🇸 spelling, [per our style guide](https://support.1password.com/style-guide/#spelling). 

```suggestion
      description: Leave blank if same as the Homepage URL. If you're part of a team, pick your favorite repo!
```